### PR TITLE
:bug: Adds polyfills to storybook

### PR DESCRIPTION
**Pull request by QuintonC**
## ℹ️ What is the context for these changes?

Adds polyfills to Storybook to ensure that QR codes properly display.

## 🕹️ Demonstration

| Before | After |
| - | - |
| Screenshot 2023-03-17 at 12 40 16 PM | Screenshot 2023-03-17 at 12 42 41 PM |

## 🎩 How can this be tophatted?

[📖  Storybook](https://639b1f308693132693d9b82c-gucxkfslez.chromatic.com/?path=/story/connect-wallet--playground)

### CODE REVIEW

File: examples/vite-custom-theme/vite.config.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {defineConfig} from 'vite';
 import react from '@vitejs/plugin-react';
-import nodePolyfills from 'vite-plugin-node-stdlib-browser';
+import {nodePolyfills} from 'vite-plugin-node-polyfills';
```
Comment (caropinzonsilva) [inline examples/vite-custom-theme/vite.config.ts:3]: Could we have used the old `nodePolyfills` in Storybook? I'm curious about what is the difference between them.

**Review by caropinzonsilva — APPROVED** [review]
Code looks good!

File: examples/vite-custom-theme/vite.config.ts (hunk 1 above)
Comment (QuintonC) [reply examples/vite-custom-theme/vite.config.ts:3]: > Could we have used the old `nodePolyfills` in Storybook? I'm curious about what is the difference between them.

We could have, yes. The main difference is that the new has type declarations (it's also the same one we use in the [example app](https://github.com/Shopify/tokengating-example-app/blob/main/extensions/tokengate-src/package.json#L22).

Comment (caropinzonsilva) [reply examples/vite-custom-theme/vite.config.ts:3]: Makes sense!

**Pull request merged**